.config/spellcheck.dic

```diff
@@ -3,3 +3,22 @@ Ethereum
 ERC20
 Wasm
 P2PKH
+P2WPKH
+millisatoshi
+vbyte
+vbytes
+UTXO
+utxos
+UTXOs
+Oisy
+PAPI
+backend
+backend's
+readonly
+pricelist
+ckBTC
++
```

Comment:
is this one required? really? eheh! just curiosity

Reply:
Yup.  It is possible to specify patterns but the syntax for that is so arcane that it isn't worth it for us.